http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/ClusterHelper.scala

```diff
@@ -428,7 +426,8 @@ class ClusterHelper(
           clusterResourcesConfig.rstudioDockerCompose,
           clusterResourcesConfig.proxyDockerCompose,
           clusterResourcesConfig.proxySiteConf,
-          clusterResourcesConfig.welderDockerCompose
+          clusterResourcesConfig.welderDockerCompose,
+          clusterResourcesConfig.jupyterNotebookConfigUri
```

Comment:
`jupyterNotebookConfigUri` is a non-templated resource now. Technically we can/should probably put it in the base Jupyter image itself. However Leo still needs to pass it to support older image versions.

Reply:
add you add a comment about this in source code?

Reply:
Yeah will do in a separate PR